src/contract.rs

```diff
@@ -178,7 +178,9 @@ pub fn execute(
                     amount: size,
                     denom: base,
                 },
-                events: vec![],
+                remaining_base: Uint128::zero(),
```

Comment:
the way this variable is named, it reads like this should start at base size, quote size, fee size and then reduce per event but i think the way it's structured, it's the opposite of that?

Reply:
I was thinking about that after I finished the PR. It would be more appropriate to call those `accumulated_{base,fee,quote}`.